.gitignore

```
@@ -216,3 +216,4 @@ libprof.so
 #Ignore everything but the example config in the media jsons directory, For easy testing.
 /config/media/jsons/*
 !/config/media/jsons/__EXAMPLE.json
+/config/_devtest.txt
```

leftover?

---

Intentional with no side effects. I can atomize it out if you'd like.

---

Y is it here

---

the devtest config was added mostly so that I can set config options to things convenient for local testing, without any chance of committing them to a public repo.

---

...Or if you meant why is it in this category, it's because that's just the end of the file.

This work for you?
```suggestion

#Local Developer config file. Don't commit this to the repo, as it's all overrides.
/config/_devtest.txt
```